### ISSUE TITLE
Dataset collection

### ISSUE THREAD

**Issue by Abdulazizbek** [axin1301/Satellite-imagery-dataset#2]
Thank you for sharing the great resource.

Is it possible to share a `code` of `data collection` process (zoom level was 19 and about 12.2 million images each year) from satellite. (For repeat the whole process from the beginning)

Thank you beforehand,

**Comment by axin1301** [comment]
Yes, of course. I will find the related files and send to you in several days. Could you provide an email address? Thank you.
________________________________
发件人: Abdulaziz ***@***.***>
发送时间: 2024年4月9日 15:41
收件人: axin1301/Satellite-imagery-dataset ***@***.***>
抄送: Subscribed ***@***.***>
主题: [axin1301/Satellite-imagery-dataset] Dataset collection (Issue #2)


Thank you for sharing the great resource.

Is it possible to share a code of data collection process (as written in the paper: zoom level was 19 and about 12.2 million images each year) from satellite.

Thank you beforehand,

―

**Comment by Abdulazizbek** [comment]
Thank you for your reply, 
varzik91@gmail.com
